packages/edit-site/src/components/home-view-preview/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+
+import Editor from '../editor';
+
+export function HomeViewPreview() {
+	const { isBlockBasedTheme, siteUrl } = useSelect( ( select ) => {
+		const { getEntityRecord, getCurrentTheme } = select( coreStore );
+		const siteData = getEntityRecord( 'root', '__unstableBase' );
+
+		return {
+			isBlockBasedTheme: getCurrentTheme()?.is_block_theme,
+			siteUrl: siteData?.home,
+		};
+	}, [] );
+
+	// If theme is block based, return the Editor, otherwise return the site preview.
+	return isBlockBasedTheme ? (
+		<Editor />
+	) : (
+		<iframe
+			src={ siteUrl }
+			title="front-end view"
+			style={ {
+				display: 'block',
+				width: '100%',
+				height: '100%',
+			} }
+			onLoad={ ( event ) => {
+				// Hide the admin bar in the front-end preview.
+				const document = event.target.contentDocument;
+				document.getElementById( 'wpadminbar' ).remove();
+				document
+					.getElementsByTagName( 'html' )[ 0 ]
+					.setAttribute( 'style', 'margin-top: 0 !important;' );
+				// Make links unclickable.
+				const links = document.getElementsByTagName( 'a' );
+				Array.from( links ).forEach( ( link ) => {
+					link.style.pointerEvents = 'none';
+				} );
```

Comment:
Is there any intention beyond making links unclickable? E.g., should everything be disabled (except scrolling?).

With `pointerEvents` links are navigable/clickable by keyboard.

Also buttons can be clicked, e.g., search bar

https://github.com/user-attachments/assets/79fe123d-acfd-4a76-9370-0ad64316fd5e

It's not such a big deal - the consequences are pretty low, and it's still reasonable accessible I'd guess. Would adding `aria-disabled="true"` to these elements help?

Reply:
Yeah they shouldn't be clickable or focusable at all. I think tabindex is our best bet for that, but probs best aria-disable it too.